apps/api/src/app/notifications/dtos/activities-response.dto.ts

```diff
@@ -173,7 +173,7 @@ export class ActivityNotificationResponseDto {
   transactionId: string;
 
   @ApiPropertyOptional()
-  createdAt?: Date;
+  createdAt?: string;
```

Comment:
I think this doesn't break anything as I think we were truly returning strings (this DTO is used in API responses and I couldn't find any transformation to Date in the code). Worth review this in the Web app. Running in my local couldn't find any problem with this change though. 😕